EvolutionModelTests/AflegstapelTests.cs

```diff
@@ -38,5 +39,39 @@ public class AflegstapelTests {
         Assert.AreEqual(cardAmount, actualCount);
       }
     }
+
+    [TestMethod]
+    public void AflegstapelIsLeegNaEmpty() {
+      var aflegstapel = new Aflegstapel();
+
+      for(int i = 0; i < 5; i++) {
+        var card = new Card();
+        aflegstapel.Add(card);
+      }
+      Assert.AreEqual(5, aflegstapel.Count);
+
+      aflegstapel.Empty();
+      Assert.AreEqual(0, aflegstapel.Count);
+    }
+
+    [TestMethod]
+    public void AflegstapelReturntAlleKaartenDieErinZatenNaEmpty() {
+      var aflegsapel = new Aflegstapel();
+      var cards = Enumerable.Range(0, 5).Select(x => new Card());
```

Comment:
Ik maak hier een sequence (IEnumerable<Card>) van 5 kaarten. Later vergelijk ik die met de kaarten die gereturnd worden.

`Range` is een static method in de class `Enumerable`. Die neemt 2 argumenten, een beginwaarde en de lengte. Wat er gereturnd wordt is een sequence met die eigenschappen. In dit geval dus `0, 1, 2, 3, 4`. `Select` maakt voor elk element in die sequence een nieuwe kaart aan. Uiteindelijk hebben we dus 5 kaarten.